corehq/apps/domain/forms.py

```diff
@@ -1696,6 +1696,8 @@ def save(self, commit=True):
     def is_downgrade(self):
```

Comment:
This function is a bit of a misnomer now.  Might be better to rename to ```is_downgrade_from_paid_plan``` or something like that.

Reply:
(Fine to address in a separate PR)

Reply:
You're right, this makes more sense